patches/minecraft/net/minecraft/world/entity/player/Player.java.patch

```diff
@@ -295,6 +302,14 @@
                          this.m_21008_(InteractionHand.MAIN_HAND, ItemStack.f_41583_);
                       }
                    }
+@@ -1215,6 +_,7 @@
+ 
+          }
+       }
++      this.cachedAttackStrength = 0.0F; // Forge: Resets the attackStrength field once the attack code is finished
```

Comment:
This reset can be removed, as it is spurious.  Whenever the value is considered "valid", it will have been set, meaning resetting it has no real benefit.